docs/HISTORY.txt

```diff
@@ -5,7 +5,9 @@ Changelog
 1.5.3 (unreleased)
 ------------------
 
-- Fix iframe auto resizer [Nachtalb]
+- Fix iframe auto resizer not being triggered on plone 4 [Nachtalb]
+- Ensure iframeResizer.contentWindow is included on Plone 5 - in case iframe
+  includes a page from the same site [Nachtalb]
```

Comment:
Would you mind putting this?
Ensure iframeResizer.contentWindow is included on Plone 5 - in case iframe includes a page from the the same site

Reply:
It is already that 😂

Reply:
"welp"